File: py/IEC62325/MarketOperations/MarketSystem/MarketResults/Settlement.py
from IEC61968.Common.Document import Document
from IEC61970.Base.Domain.DateTime import DateTime


class Settlement(Document):
	"""
	#  * Specifies a settlement run.
	#  * @created 28-Dec-2023 8:26:27 PM
	"""
	def __init__(self):
		super().__init__()
		# # 	 * The trade date on which the settlement is run.
		self.trade_date = DateTime()

File: java/IEC62325/MarketOperations/MarketSystem/MarketResults/Settlement.java
package IEC62325.MarketOperations.MarketSystem.MarketResults;

import IEC61970.Base.Domain.DateTime;
import IEC61968.Common.Document;

/**
 * Specifies a settlement run.
 * @created 03-Jan-2024 2:09:38 PM
 */
public class Settlement extends Document {

	/**
	 * The trade date on which the settlement is run. 
	 */
	public DateTime tradeDate;

	public Settlement(){

	}
}//end Settlement